### PR TITLE
Fix Catalina bug

### PR DESCRIPTION
**Pull request by jbmouret**
see #75

### CODE REVIEW

File: src/python/example.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def __call__(self):
             self._states.append(self._simu.robot(0).positions())
 
 # Load robot from URDF
-robot = rd.Robot("res/models/arm.urdf", "arm", False)
+robot = rd.Robot("res/robots/arm.urdf", "arm", False)
```
Comment (costashatz) [inline src/python/example.py:33]: I guess these are left-overs from another branch right?

File: src/python/example_parallel.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 from timeit import default_timer as timer
 import os
 
-robot = rd.Robot("res/models/arm.urdf", "arm", False)
+robot = rd.Robot("res/robots/arm.urdf", "arm", False)
```
Comment (costashatz) [inline src/python/example_parallel.py:8]: > I guess these are left-overs from another branch right?

Same here.

Comment (jbmouret) [reply src/python/example_parallel.py:8]: Yes. Sorry, I had a few things staged and I wrote 'commit' a bit too fast. This is from urdf_library...

Comment (costashatz) [reply src/python/example_parallel.py:8]: Can you revert them? (or just do a new commit that is undoing them)

**Comment by costashatz** [comment]
**Merging..**

**Pull request merged**